pkg/server/handler.go

```diff
@@ -209,6 +211,11 @@ func WithWorkspaceProjection(apiHandler http.Handler) http.HandlerFunc {
 			return
 		}
 
+		if cluster.Name == tenancyv1alpha1.RootCluster && req.URL.Path == getHomeWorkspaceRequestPath {
```

Comment:
what is this doing here?

Reply:
BTW fixed in commit a1e36e31d467daafd64d6cf57f61268794951ea3